sale_comment_template/readme/CONTRIBUTORS.rst

```diff
@@ -6,5 +6,6 @@
 
   * Pedro M. Baeza <pedro.baeza@tecnativa.com>
   * Vicent Cubells <vicent.cubells@tecnativa.com>
+* `DynApps <https://www.dynapps.be>`_:
 
-Do not contact contributors directly about support or help with technical issues.
```

Comment:
No need to remove this sentence

Reply:
This sentence is automatically added when generating the README.rst

Reply:
No, it isn't... It would be good any way if you pregenerate the README for having all included (and you will see that it's not included). It would be advisable any way to include it on the template generator.

Reply:
IT is included!

![image](https://user-images.githubusercontent.com/8638295/54909033-86f41380-4ee9-11e9-9b32-
a0357853d321.png)

Reply:
OK, I see! It's included in other section: https://github.com/OCA/maintainer-tools/blob/2b5b85b0bdc8515a38a1b9ce4056cf998c91f793/tools/gen_addon_readme.template#L48

Reply:
I always pre-generate the README file ;-)